README.md

```diff
@@ -1 +1,25 @@
-# mysqld_exporter (UNDER CONSTRUCTION)
+# mysqld_exporter
+
+Exporter for MySQL server metrics http://prometheus.io/
+
+## Building and running
+
+    make
+    ./mysqld_exporter <flags>
+
+## Configuration
+The configuration is in JSON. An example with all possible options:
```

Comment:
Yeah, password would be a problem then.

Can we make the JSON structure simpler? I think config can be removed